spring-cloud-aws-sqs/src/main/java/io/awspring/cloud/sqs/config/SqsBeanNames.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2013-2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.awspring.cloud.sqs.config;
+
+/***
+ * Utility class containing the bean names used for the framework's bean registration.
+ *
+ * @author Tomaz Fernandes
+ * @since 3.0
+ */
+public class SqsBeanNames {
+
+	private SqsBeanNames() {
+	}
+
+	/**
+	 * The bean name of the {@link io.awspring.cloud.sqs.listener.DefaultListenerContainerRegistry} registered by
+	 * {@link SqsBootstrapConfiguration}.
+	 */
+	public static final String ENDPOINT_REGISTRY_BEAN_NAME = "io.awspring.cloud.messaging.internalEndpointRegistryBeanName";
+
+	/**
+	 * The bean name of the {@link io.awspring.cloud.sqs.annotation.SqsListenerAnnotationBeanPostProcessor} registered
+	 * by {@link SqsBootstrapConfiguration}.
+	 */
+	public static final String SQS_LISTENER_ANNOTATION_BEAN_POST_PROCESSOR_BEAN_NAME = "io.awspring.cloud.messaging.internalSqsListenerAnnotationBeanPostProcessor";
```

Comment:
Move those to `SqsBootstrapConfiguration` file given that is the only place where those are used.

Reply:
I think the reasoning for this class is that these bean names are part of the `public API`, and this way users can use the constants without having to dig into `SqsBootstrapConfiguration` - seems like a valid reason for me.

That being said, I'm ok with moving it as well.

WDYT?